webapp/lib/controller.dart

```diff
@@ -48,10 +66,12 @@ void init() async {
 void initUI() {
   window.location.hash = '#/dashboard'; //TODO This is just temporary initialization becuase we don't have a complete app
   router.routeTo(window.location.hash);
+  view.navView.projectTitle = 'COVID IMAQAL Batch replies (Week 12)'; //TODO To be replaced by data from model
```

Comment:
This is the package title not the project title. I think here it should read just 'COVID IMAQAL', and then in `loadConfigurationView` the title should be rewritten as 'COVID IMAQAL Batch replies (Week 12)' (with the todo to be replaced with data from the model)

Reply:
CHanged it. I'll do the rewriting when I address issues raised in the review of #15 so as to reduce the number of merge conflicts